### DISCUSSION TITLE
Sub CA for the deployment

### DISCUSSION THREAD

**Discussion by Meritadev** [vexxhost/atmosphere#30] [Q&A] [answered]
we have a question related to the PKI, if we would like to rely on existing CA, will it be enough to have Sub CA for the deployment?

**Comment by okozachenko1203** [comment, marked as answer]
Atmosphere supports any kind of issuer spec.
So in this case, we can use CA issuer with the sub CA provided from customer

**Reply by mnaser** [reply]
That's correct.  Indeed, we already use the built-in CA issuer to generate self-signed certificate in our Molecule deployment.  I think the only thing missing here is the actual documentation on how to do this.

I've created an issue for building documentation on how to do this here: https://github.com/vexxhost/atmosphere/issues/31
